crawler/cmd/one.go

```diff
@@ -30,8 +31,13 @@ var oneCmd = &cobra.Command{
 
 		c := crawler.NewCrawler(dryRun)
 
-		repoURL, whitelists := args[0], args[1:]
-		err := c.CrawlRepo(repoURL, getPAfromWhiteList(repoURL, whitelists))
+		whitelists := args[1:]
+		url, err := url.Parse(args[0])
+		if err != nil {
+			log.Error(err)
```

Comment:
maybe we should consider to stop the program if these kind of errors are raised.

Reply:
Fixed in the `code_refactoring` branch.